ci-operator/config/openshift/cluster-openshift-apiserver-operator/openshift-cluster-openshift-apiserver-operator-master.yaml

```diff
@@ -37,14 +37,23 @@ tests:
 - as: verify
   commands: |
     go get -u golang.org/x/tools/imports
-    make verify -k
+    make verify -k --warn-undefined-variables
   container:
     from: bin
 
+- as: verify-deps
+  commands: |
+    HOME=$( mktemp -d )
+    go get -u github.com/Masterminds/glide
```

Comment:
this is not a path to happiness.  it is going to limit adoption if we have these fix ups lying around.

Reply:
I think this was meant for HOME line, and I agree, just wanted to make it work without CI - temporary workaround

Reply:
@deads2k Issue filled, we can move with the workaround in the meantime
https://github.com/openshift/release/issues/2057